cellrank/pl/_gene_trend.py

```diff
@@ -134,6 +134,8 @@ def gene_trends(
         Whether to show gene names as titles instead on y-axis.
     legend_loc
         Location of the legend displaying lineages. Only used when `same_plot=True`.
+    obs_legend_loc
```

Comment:
I agree, let's use `lineage_legend_loc` and `obs_legend_loc`.
The only alternative would be to merge `cbar` and `obs_legend_loc` into 1 argument, since they are mutually exclusive, though I am not sure what the name would be.

Reply:
But `cbar` is `bool`, no? Let's leave them separate I would say.